chef/cookbooks/provisioner/templates/suse/crowbar_register.erb

```diff
@@ -22,7 +22,7 @@
 
 usage () {
     cat <<EOF
-`basename $0` [-h|--help] [-v|--verbose] [-f|--force] [--gpg-auto-import-keys] [--no-gpg-checks] [--interface IF]
+`basename $0` [-h|--help] [-v|--verbose] [-f|--force] [--gpg-auto-import-keys] [-k|--keep-existing-hostname] [--no-gpg-checks] [--interface IF]
```

Comment:
I'm coming after the merge, but still some comments: maybe we should drop the option when `node[:provisioner][:keep_existing_hostname]` is set? (It doesn't make a lot of sense in that case). Or maybe this option should accept a 0/1 value?